src/praisonai/tests/integration/test_w1_real_agentic.py

```
@@ -0,0 +1,95 @@
+        agent, "tg-12345",
+        "Remember this: my favourite colour is octarine.",
+    )
+    print("\n[Telegram in] my favourite colour is octarine")
+    print(f"[Telegram out] {fact_response}\n")
+
+    # 6. Ask via "Discord" — agent should recall from unified history
+    recall_response = await mgr_d.chat(
+        agent, "dc-67890",
+        "What did I just tell you my favourite colour was?",
+    )
+    print(f"[Discord in] What did I just tell you my favourite colour was?")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)